pages/common/gox.md

```diff
@@ -0,0 +1,19 @@
+# gox
+
+> A tool for Go cross compilation.
+
+- Compile current Go directory for all OS and arch:
```

Comment:
I assume Go programs are laid out in a specific directory structure?

Perhaps this will read better?
```suggestion
- Compile Go program in the current directory for all operating systems and architecture combinations:
```